src/payment/strategies/paypal/paypal-express-payment-strategy.spec.ts

```diff
@@ -51,6 +51,7 @@ describe('PaypalExpressPaymentStrategy', () => {
                 CARD: 'card',
                 CREDIT: 'credit',
             },
+            Buttons: jest.fn(),
```

Comment:
Does this PR affecting PayPal Express payment strategy also?

Reply:
Updated PayaplSDK interface. This method is not used in express